source/Calamari.GoogleCloudAccounts/GoogleCloudAccountExtensions.cs

```diff
@@ -0,0 +1,315 @@
+using System;
```

Comment:
This has been copied from the GoocleCloudContextScriptWrapper and is functionally identical.